org.eclipse.jdt.core/pom.xml

```diff
@@ -230,32 +230,70 @@
 
   <groupId>org.aspectj</groupId>
   <artifactId>org.eclipse.jdt.core</artifactId>
-  <version>3.25.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
+  <version>1.9.7-SNAPSHOT</version>
+
+  <name>JDT Core for AspectJ</name>
+  <description>
+    AspectJ forks Eclipse Java Development Tools (JDT) Core, utilising the Eclipse Compiler (ECJ) + APT classes as the
+    foundation for the AspectJ Compiler (AJC).
+  </description>
+  <url>https://www.eclipse.org/aspectj/</url>
+
+  <licenses>
+    <license>
+      <name>Eclipse Public License - v 2.0</name>
+      <url>https://www.eclipse.org/legal/epl-v20.html</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
```

Comment:
I think we probably should include EPL v1 as well. The AspectJ extensions to JDT Core are not EPL v2, because we haven't sought clearance from the original developers that it is ok to relicense them.

Reply:
OK, I am going to do that. First I wanted to use v1, but then saw that the original uses v2 and was not sure what to do, so I thought it would be safer to accept their own licence. If we can add both, I shall do so. I am not knowledgeable in that area.

Reply:
Done. It will become effective for the next snapshot, milestone and final releases, but of course it was too late for 1.9.2.M2.